app.js

```diff
@@ -51,6 +51,21 @@ global.app.use(require('compression')());
 
 // Cookies
 global.app.use(require('cookie-parser')());
+global.app.use(require('express-session')({
+    secret: (function() {
+        var d = new Date().getTime();
+        return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+            var r = (d + Math.random() * 16) % 16 | 0;
+            d = Math.floor(d / 16);
+            return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
+        });
+    })(),
+  resave: false,
+  saveUninitialized: true
+}));
+
+var auth = require('./core/auth')(app);
```

Comment:
require it next to `app.use(auth.everyauth.middleware());`, where all middlewares are